ZKSyncSDK/Classes/Model/Transactions/ChangePubKey.swift

```diff
@@ -18,18 +18,21 @@ public class ChangePubKey: ZkSyncTransaction {
     let feeToken: UInt16
     let fee: String
     let nonce: UInt32
+    let timeRange: TimeRange
+    
     var signature: Signature?
-    var ethSignature: String?
+    var ethAuthData: T?
     
     var feeInteger: BigUInt { BigUInt(fee)! }
     
-    public init(accountId: UInt32, account: String, newPkHash: String, feeToken: UInt16, fee: String, nonce: UInt32) {
+    public init(accountId: UInt32, account: String, newPkHash: String, feeToken: UInt16, fee: String, nonce: UInt32, timeRange: TimeRange) {
```

Comment:
The same goes here. The 99% of the users will not need to set timestamps on the transactions. They should be able to forget about them and not supply them when they don't need it

Reply:
Done. For all different types of transactions.